std/bytes/mod.ts

```diff
@@ -98,36 +98,47 @@ export function repeat(origin: Uint8Array, count: number): Uint8Array {
   }
 
   if (count < 0) {
-    throw new Error("bytes: negative repeat count");
+    throw new RangeError("bytes: negative repeat count");
   } else if ((origin.length * count) / count !== origin.length) {
-    throw new Error("bytes: repeat count causes overflow");
+    throw new RangeError("bytes: repeat count causes overflow");
   }
 
   const int = Math.floor(count);
 
   if (int !== count) {
-    throw new Error("bytes: repeat count must be an integer");
+    throw new TypeError("bytes: repeat count must be an integer");
   }
 
   const nb = new Uint8Array(origin.length * count);
 
-  let bp = copyBytes(origin, nb);
+  let bp = copy(origin, nb);
 
   for (; bp < nb.length; bp *= 2) {
-    copyBytes(nb.slice(0, bp), nb, bp);
+    copy(nb.slice(0, bp), nb, bp);
   }
 
   return nb;
 }
 
-/** Concatenate two binary arrays and return new one.
- * @param origin origin array to concatenate
- * @param b array to concatenate with origin
+/** Concatenate multiple binary arrays and return new one.
+ * @param origin binary array to concatenate
+ * @param buf binary arrays to concatenate with origin
  */
-export function concat(origin: Uint8Array, b: Uint8Array): Uint8Array {
-  const output = new Uint8Array(origin.length + b.length);
-  output.set(origin, 0);
-  output.set(b, origin.length);
+export function concat(a: Uint8Array, ...buf: Uint8Array[]): Uint8Array {
```

Comment:
I believe there should be only a single argument to this function (`...buf`)

Reply:
Sounds good to me as well. I will make the changes.